https://bugs.gentoo.org/508852
https://gcc.gnu.org/PR61164

2014-04-27  Magnus Granberg  <zorry@gentoo.org>

	#508852
	* libitm/configure.tgt: Disable FORTIFY

--- a/libitm/configure.tgt
+++ b/libitm/configure.tgt
@@ -43,6 +43,16 @@ if test "$gcc_cv_have_tls" = yes ; then
   esac
 fi
 
+# FIXME: error: inlining failed in call to always_inline 
+# ‘int vfprintf(FILE*, const char*, __va_list_tag*)’
+# : function body can be overwritten at link time
+# Disable Fortify in libitm for now. #508852
+case "${target}" in
+    *-*-linux*)
+	XCFLAGS="${XCFLAGS} -U_FORTIFY_SOURCE"
+	;;
+esac
+
 # Map the target cpu to an ARCH sub-directory.  At the same time,
 # work out any special compilation flags as necessary.
 case "${target_cpu}" in
